### PR TITLE
[Feat] - Update Contributing Doc, Readme and Add Code of Conduct

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

This PR,

- Updates the README.md
- Moved Contributing Flow to Separate File
- Added Code of Conduct

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Contributing Overview
```
Comment (alabulei1) [inline CONTRIBUTING.md:1]: Can we specify that this contributing guide works for WasmEdge/docs repo?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+- <a name="contributing-flow">Contribution Flow</a>
+  - <a href="#commit-signoff">Developer Certificate of Origin</a>
+- WasmEdge Contribution Flow
+  - <a href="#contributing-docs">WasmEdge Docs</a> 
+  - <a href="https://wasmedge.org/book/en/contribute/contribute.html">WasmEdge Runtime</a>
+  - <a href="https://github.com/WasmEdge/WasmEdge/blob/master/SECURITY.md">Reporting a vulnerability</a>
+
+# <a name="contributing-flow">Contribution Flow</a>
+
+To contribute to WasmEdge, have a look at [here](https://wasmedge.org/docs/contribute/overview)
```
Comment (alabulei1) [inline CONTRIBUTING.md:20]: Contributing to WasmEdge is different from contributing to WasmEdge docs. Let's specify the goal of this contributing guide.

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
Hi @adithyaakrishna 

Thanks for your contribution. I have left some feedback. Let me know what do you think.

File: CONTRIBUTING.md (hunk 2 above)
Comment (adithyaakrishna) [reply CONTRIBUTING.md]: Oops! My bad, thanks for pointing it out :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (adithyaakrishna) [reply CONTRIBUTING.md]: Done 💯

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+## <a name="contributing-docs">WasmEdge Docs</a> 
+
+If you want to contribute to the WasmEdge Docs, you can follow these steps:
+
+1. **Fork the project:** Click the "Fork" button on the top-right corner of the repository page to create a copy of the project in your GitHub account.
+
+2. **Clone the forked project:** Open a terminal or command prompt and type the following command:
+
+```
+git clone <forked-repository-url>
```
Comment (alabulei1) [inline CONTRIBUTING.md:69]: Can we just say `git clone https://github.com/WasmEdge/docs.git`? In my mind, the contribution flow here just describes how to contribute to WasmEdge docs. The contribution flow for WasmEdge runtime will be much difficult.  I'd love to hear what you think.

Comment (adithyaakrishna) [reply CONTRIBUTING.md:69]: Yes, it makes complete sense with respect to WasmEdge Runtime, but for this line, we are referring to the user's forked repository URL, and adding the `https://github.com/WasmEdge/docs.git` IMO wouldn't make much sense here. But please feel free to let me know if this is something the team wants and I will gladly send a commit to fix it :)

**Pull request merged**